### ISSUE TITLE
GWT is broken

### ISSUE THREAD

**Issue by SuperBadger** [TomGrill/gdx-facebook#36]
html:compileGwt always fails, everything installed according to wiki instructions:

```
/gdx-facebook-core-1.2.2-sources.jar!/de/tomgrill/gdxfacebook/c
ore/utils/Utils.java'
         [ERROR] Line 46: No source code is available for type java.net.URL; did
 you forget to inherit a required module?
         [ERROR] Line 72: No source code is available for type java.net.Malforme
dURLException; did you forget to inherit a required module?
         [ERROR] Line 81: No source code is available for type java.net.URLDecod
er; did you forget to inherit a required module?
   [ERROR] Aborting compile due to errors in some input files
:html:compileGwt FAILED
```

**Comment by TomGrill** [comment]
cannot reproduce. Please create a simple libGDX project which causes this error.

**Comment by aryan9234** [comment]
Compiled in super dev mode and debug perfectly, this error occur only when i compile in dist mode for distribution.

**Comment by TomGrill** [comment]
fixed with 1.3.0 coming in few hours
